### PR TITLE
Add Image title

### PR DESCRIPTION
**Pull request by igorbrasileiro**
#### What problem is this solving?
Add Image title back removed here https://github.com/vtex-apps/store-image/pull/54

#### How to test it?

[Workspace](https://brasileiro--storecomponents.myvtex.com)

#### Related to / Depends on
https://github.com/vtex-apps/render-runtime/pull/627

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
     "vtex.device-detector": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.device-detector@0.2.6/public/@types/vtex.device-detector",
     "vtex.list-context": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.list-context@0.2.0/public/@types/vtex.list-context",
-    "vtex.native-types": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.native-types@0.7.4/public/@types/vtex.native-types",
+    "vtex.native-types": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.native-types@0.7.5/public/@types/vtex.native-types",
     "vtex.on-view": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.on-view@1.0.0/public/@types/vtex.on-view",
     "vtex.pixel-manager": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.pixel-manager@1.8.0/public/@types/vtex.pixel-manager",
-    "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.132.2/public/@types/vtex.render-runtime",
+    "vtex.render-runtime": "https://brasileiro--storecomponents.myvtex.com/_v/private/typings/linked/v1/vtex.render-runtime@8.132.2+build1633464084/public/@types/vtex.render-runtime",
```
Comment (lariciamota) [inline react/package.json]: Remember to update this afterward

**Review by lariciamota — APPROVED** [review]

**Review by victorhmp — APPROVED** [review]

**Comment by igorbrasileiro** [comment]
@victorhmp @lariciamota, take a look here again.

**Review by victorhmp — APPROVED** [review]

File: react/Image.tsx [hunk 2]
```diff
@@ -154,9 +154,16 @@ function Image(props: ImageProps) {
         }
       : undefined
 
+  const formatedLink = formatIOMessage({ id: link?.url, intl })
+  const formatedTitle = formatIOMessage({ id: link?.attributeTitle, intl })
```
Comment (lariciamota) [inline react/Image.tsx]:
```suggestion
  const formattedLink = formatIOMessage({ id: link?.url, intl })
  const formattedTitle = formatIOMessage({ id: link?.attributeTitle, intl })
```

File: react/Image.tsx [hunk 3]
```diff
@@ -154,9 +154,16 @@ function Image(props: ImageProps) {
...
+
   const maybeLink = link?.url ? (
+    // The onclick function and the <Link> component are necessary,
+    // so knowing this disable the anchor-is-valid rule seems good.
```
Comment (lariciamota) [inline react/Image.tsx]:
```suggestion
    // The onClick function and the <Link> component are necessary,
    // knowing this, it seems good to disable the anchor-is-valid rule.
```

File: react/ImageSlider.tsx [hunk 4]
```diff
@@ -43,24 +43,46 @@ function ImageSlider(props: ImageSliderProps) {
     <SliderLayout {...sliderLayoutConfig} totalItems={images.length}>
       {images.map(
         ({ image, mobileImage, link, description, ...otherProps }, idx) => {
+          const formatedImage = formatIOMessage({ id: image, intl })
+          const formatedMobileImage = formatIOMessage({ id: mobileImage, intl })
```
Comment (lariciamota) [inline react/ImageSlider.tsx]:
```suggestion
          const formattedImage = formatIOMessage({ id: image, intl })
          const formattedMobileImage = formatIOMessage({ id: mobileImage, intl })
```

**Pull request merged**